docs/updating_schemas.md

```diff
@@ -216,6 +220,8 @@ Exception: When making substantial changes to an existing schema, even if these
 * **NOTE:** You should update the schema version when making substantial changes to an existing schema even if these changes do not cause existing objects using this schema to fail validation.
 The new schema version number in such cases helps submitters and users realize that a substantial change has been made to the schema and they may need to update their scripts accordingly.
 
+* **NOTE:** You should also update the appropriate changelog markdown file with documentation of the new schema version, in **schemas/changelogs** directory.  You should update the changelog even if the schema version number has not been bumped up. If so, it appears as "minor changes since schema version xyz".  See changelogs in [encoded](https://github.com/ENCODE-DCC/encoded) for an example.
```

Comment:
Kalina, I don't think you should refer to ENCODED from this repo.

Reply:
Done